examples/fluids/src/setuplibceed.c

```diff
@@ -243,14 +270,20 @@ PetscErrorCode CreateOperatorForDomain(Ceed ceed, DM dm, SimpleBC bc,
                            &op_apply_outflow_jacobian);
         CeedOperatorSetField(op_apply_outflow_jacobian, "dq", elem_restr_q_sur,
                              ceed_data->basis_q_sur, CEED_VECTOR_ACTIVE);
+        CeedOperatorSetField(op_apply_outflow_jacobian, "Grad_dq", elem_restr_q_sur,
+                             ceed_data->basis_q_sur, CEED_VECTOR_ACTIVE);
         CeedOperatorSetField(op_apply_outflow_jacobian, "surface qdata",
                              elem_restr_qd_i_sur,
                              CEED_BASIS_COLLOCATED, q_data_sur);
+        CeedOperatorSetField(op_apply_outflow_jacobian, "x", elem_restr_x_sur,
+                             ceed_data->basis_x_sur, ceed_data->x_coord);
         CeedOperatorSetField(op_apply_outflow_jacobian, "surface jacobian data",
                              elem_restr_jd_i_sur,
                              CEED_BASIS_COLLOCATED, jac_data_sur);
         CeedOperatorSetField(op_apply_outflow_jacobian, "v", elem_restr_q_sur,
                              ceed_data->basis_q_sur, CEED_VECTOR_ACTIVE);
+        CeedOperatorSetField(op_apply_outflow_jacobian, "dummy", elem_restr_q_sur,
+                             ceed_data->basis_q_sur, CEED_VECTOR_ACTIVE);
```

Comment:
We can get rid of this now, right?

Reply:
#988 is merged so I'd suggest just cherry-picking the above. Are you planning on a squash-merge or doing some light squashing in this branch's history?